src/infrastructure/available-data/available-data.test.ts

```diff
@@ -0,0 +1,126 @@
+import "reflect-metadata";
+import { AvailableDataStore, AvailableDataType } from "../../topics/attester";
+import { MemoryAvailableDataStore } from "./memory-available-data";
+import { LocalAvailableDataStore } from "./local-available-data";
+
+enum TestType {
+  Memory,
+  Local,
+}
+
+type TestData = {
+  store: AvailableDataStore;
+};
+
+const testAvailableData: { [name: string]: AvailableDataType } = {
+  attester1_0: {
+    attesterName: "attester1",
+    timestamp: 1,
+    metadata: {
+      url: "https://1-0",
+    },
+  },
+  attester1_1: {
+    attesterName: "attester1",
+    timestamp: 2,
+    metadata: {
+      url: "https://1-1",
+    },
+  },
+  attester2_0: {
+    attesterName: "attester2",
+    timestamp: 3,
+    metadata: {
+      url: "https://2-1",
+    },
+  },
+};
+
+describe("test available data", () => {
+  const testCases = [[TestType.Memory], [TestType.Local]];
+  let testData: { [name in TestType]: TestData };
+
+  beforeEach(() => {
+    const localStore = new LocalAvailableDataStore("test-available-data");
+    localStore.reset();
+    testData = {
+      [TestType.Memory]: {
+        store: new MemoryAvailableDataStore(),
+      },
+      [TestType.Local]: {
+        store: localStore,
+      },
+    };
+  });
+
+  it.each(testCases)(
```

Comment:
Why we test in Memory and in Local in the same time?

Reply:
We expect the same behaviour for local and memory. Why not test in the same time ?